errors/no-before-interactive-script-outside-document.md

```diff
@@ -0,0 +1,33 @@
+# beforeInteractive Script component outside \_document.js
+
+#### Why This Error Occurred
+
+You can't use the `next/script` component with the `beforeInteractive` strategy outside the `_document.js` page. That's because `beforeInteractive` strategy only works inside **\_document.js** and is designed to load scripts that are needed by the entire site (i.e. the script will load when any page in the application has been loaded server-side).
+
+#### Possible Ways to Fix It
+
+If you want a global script, move the script inside `_document.js` page.
+
+```jsx
+// In _document.js
+
+export default class MyDocument extends Document {
```

Comment:
Same here, should we use a functional `_document` here as that is what we've been moving docs towards https://nextjs.org/docs/advanced-features/custom-document